webpack.common.config.js

```diff
@@ -25,6 +25,7 @@ var entry = {
     // JS
     'base-page': staticPath('js/pages/base-page.js'),
     'home-page': staticPath('js/pages/home-page.js'),
+    'landing-page': staticPath('js/pages/landing-page.js'),
```

Comment:
Nevermind, a decision was just made to remove /home. https://openscience.atlassian.net/browse/OSF-5605